ArduCopter/mode_auto.cpp

```diff
@@ -821,7 +887,7 @@ void Copter::ModeAuto::spline_run()
         // get pilot's desired yaw rat
         target_yaw_rate = get_pilot_desired_yaw_rate(channel_yaw->get_control_in());
         if (!is_zero(target_yaw_rate)) {
-            auto_yaw.set_mode(AUTO_YAW_HOLD);
+            auto_yaw.set_mode(AUTO_YAW_HOLD);            
```

Comment:
Wrong change.

Reply:
The ENB_CRS_LOCK behavior is eliminated because this function is performed by the WP_YAW_BEHAVIOR parameter